HOOKS.md

```diff
@@ -1,24 +1,24 @@
+The Addon-operator pursues an agreement stating that the information is transferred to hooks via files and results of hooks execution are also stored in files. Paths to files are passed via environment variables. The output to stdout will be written to the log, except for the case with the configuration output (run with `--config` flag). Such an agreement simplifies the work with the input data and reporting the results of the hook execution.
 
 # Initialization of global hooks
 
-Global hooks can read and modify values in the global values storage which is available to all modules (see [VALUES](VALUES.md)). The global hook performs actions and discovers the values required by several modules.
+Global hooks can read and modify values in the storage of global values. Global values are also available to all modules (see [VALUES](VALUES.md)). The global hook performs actions and discovers the values required by several modules.
 
 Global hooks are stored in the $GLOBAL_HOOKS_DIR directory. The Addon-operator recursively searches all executable files in it and runs them with the `--config` flag. Each hook prints its events binding configuration in the JSON format to stdout. If the execution fails, the Addon-operator terminates with the code of 1.
 
-Bindings from [shell-operator](https://github.com/flant/shell-operator) are available for global hooks: [onStartup](#onstartup), [schedule](#schedule) and [onKubernetesEvent](#onkubernetesevent). The bindings to the events of the modules discovery process are also available: [beforeAll](#beforeall) and [afterAll](#afterall) (see [modules discovery](LIFECYCLE.md#modules-discovery)).
+Bindings from [shell-operator](https://github.com/flant/shell-operator) are available for global hooks: [onStartup](#onstartup), [schedule](#schedule) and [kubernetes](#kubernetes). The bindings to the events of the modules discovery process are also available: [beforeAll](#beforeall) and [afterAll](#afterall) (see [modules discovery](LIFECYCLE.md#modules-discovery)).
 
 # Initialization of module hooks
 
-Module hooks can get values from the global values storage. Also, they can read and modify values in the module values storage. For details on values storage, see [VALUES](VALUES.md). Hooks are initialized for all enabled modules within the process of [modules discovery](LIFECYCLE.md#modules-discovery).
+Module hooks receive values from the storage of global values. Also, they can read and modify values in the storage of module values. For details on values storage, see [VALUES](VALUES.md). Hooks are initialized for all enabled modules within the 'modules discover' process (see [modules discovery](LIFECYCLE.md#modules-discovery)).
```

Comment:
```suggestion
Module hooks receive values from the global values' storage. Also, they can read and modify values in the module values' storage. For details on values storage, see [VALUES](VALUES.md). Hooks are initialized for all enabled modules within the 'modules discover' process (see [modules discovery](LIFECYCLE.md#modules-discovery)).
```

Reply:
Storage is not significant here. This section is rewritten to explain 'module values changed' event.